nested.tmux

```diff
@@ -14,6 +14,10 @@ if [ -z "$up_recursive_keybind" ]; then up_recursive_keybind='F3'; fi
 
 inactive_status_style="$(tmux show-option -gqv @nested_inactive_status_style)"
 if [ -z "$inactive_status_style" ]; then inactive_status_style='fg=colour245,bg=colour238'; fi
+
+inactive_status_style_target="$(tmux show-option -gqv @nested_inactive_status_style_target)"
+if [ -z "$inactive_status_style_target" ]; then inactive_status_style_target='status-style'; fi
```

Comment:
Can we rename this variable to `status_style_option`? 🙏

I would drop `inactive` since it's used to go from active to inactive but also the other way around. Also `option` is the official name for this object in tmux.